docs/05-storage-providers.md

```diff
@@ -166,225 +174,241 @@ Amazon charges slightly more for non-US buckets. Known bucket locations:
+* `--force-digest-authentication`
+Using the HTTP Digest authentication method allows the user to authenticate with the server, without sending the password in clear. However, a man-in-the-middle attack is easy, because the HTTP protocol specifies a fallback to Basic authentication, which will make the client send the password to the attacker. Using this flag, the client does not accept this, and always uses Digest authentication or fails to connect.
+* `--use-ssl`
+Use this flag to communicate using Secure Socket Layer (SSL) over http (https).
 * `--debug-propfind-file`
-To aid in debugging issues, it is possible to set a path to a file that will be overwritten with the PROPFIND response  
+To aid in debugging issues, it is possible to set a path to a file that will be overwritten with the PROPFIND response
 
 ## Amazon Cloud Drive
 
-This backend can read and write data to Amazon Cloud Drive. Supported format is
+This backend can read and write data to Amazon Cloud Drive.
+
+URL Format:
 
 `amzcd://folder/subfolder`
 
-Supported options:
+Options:
 
-* `--authid (Password)`  
-The authorization code.  
+* `--authid (Password)`
+The authorization code.
 The authorization token retrieved from https://duplicati-oauth-handler.appspot.com?type=amzcd
-* `--amzcd-labels (String)`  
-The labels to set.  
-Use this option to set labels on the files and folders created  
+* `--amzcd-labels (String)`
+The labels to set.
+Use this option to set labels on the files and folders created
 Default value: `duplicati,backup`
-* `--amzcd-consistency-delay (Timespan)`  
-The consistency delay.  
-Amazon Cloud drive needs a small delay for results to stay consistent.  
+* `--amzcd-consistency-delay (Timespan)`
+The consistency delay.
+Amazon Cloud drive needs a small delay for results to stay consistent.
 Default value: `15s`
 
 ## Azure blob
 
-This backend can read and write data to Azure blob storage. Allowed format:
+This backend can read and write data to Azure blob storage.  To configure this backend you must create a storage account in the [Azure Portal](https://portal.azure.com) and get an access key by going to `Settings > Access keys` on the storage account details page.  The configuration options can either be provided as part of the URL or as separate parameters.  If both are provided, the URL parameters will take precedence.
 
-`azure://bucketname`
+URL Format:
 
-Supported options:
+`azure://containerName`
+`azure://storageAccountName@containerName`
+`azure://storageAccountName:accessKey@containerName`
 
-* `--azure_account_name (String)`  
-The storage account name.  
-The Azure storage account name which can be obtained by clicking the "Manage Access Keys" button on the storage account dashboard.
-* `--azure_access_key (Password)`  
-The access key.  
-The Azure access key which can be obtained by clicking the "Manage Access Keys" button on the storage account dashboard.
-* `--azure_blob_container_name (String)`  

```

Comment:
Currently, the provided URL is required to have a host property (mapping to the container name).  Additionally, while this parameter is defined in the code it's never used.  This parameter should either be removed (or have the code changed it can actually be used), but in the meantime I just removed this documentation.